Commands_Functions/create_command.py

```diff
@@ -0,0 +1,32 @@
+import os, json
+from Commands_Functions.abstract_command import AbtractCommand
+from schema_keys import Keys
+from output.exceptions import MissingInput, WrongInput
+from output.status import Status
+
+class CreateCommand(AbtractCommand):
+    def __init__(self, schema_path):
+        self.schema_data = CreateCommand.__validate(schema_path)
+
+    @staticmethod  
+    def __validate(schema_path):
```

Comment:
Validate expects to return true/false or throws an exception. I would rename this method by `__validate_and_get_schema`